olive/passes/pytorch/torch_trt_conversion.py

```diff
@@ -0,0 +1,177 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+import torch
+
+from olive.common.utils import tensor_data_to_device
+from olive.hardware.accelerator import AcceleratorSpec, Device
+from olive.model import PyTorchModel
+from olive.passes import Pass
+from olive.passes.olive_pass import PassConfigParam
+from olive.passes.pytorch.sparsegpt_utils import (
+    _get_attr,
+    get_layer_submodules,
+    get_layers,
+    seqlens,
+    validate_min_max_layers,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class TorchTRTConversion(Pass):
```

Comment:
>TorchTRTConversion [](http://example.com/codeflow?start=6&length=18)

does this pass support any type PT model of just some models defined in the layer map? Do we need add document to demonstrate what type of models this pass support?

Reply:
Done. Added to the docs and docstring. 
I have plans to support non hf_config pytorch models in the future. But for now, it only supports PyTorch model with hf_config and some model types.